test/api.spec.js

```diff
@@ -20,8 +20,58 @@ describe('api', () => {
   });
 
   describe('Test Recipe Paths', () => {
+    describe('GET api/v1/recipes/calorie_search?q=calorie_count', () => {
+      beforeAll( async() => {
+        for( let i = 1; i <= 6; i++){
```

Comment:
I haven't seen this notation before, but what I think I understand is going on here is you're passing a loop which tests the range of values from 1 through 6 and telling it to increment by one on each successive loop?